manifests/service.pp

```diff
@@ -7,7 +7,7 @@
 
   if $nginx::service_config_check {
     exec { 'nginx_config_check':
-      command     => 'nginx -t',
+      command     => "${nginx::service_name} -t",
```

Comment:
The command doesn't have to be the same as the service, right? For example, it may be possible that someone has `nginx.service@myinstance` if the run multiple services using systemd. Perhaps it deserves its own parameter?

Reply:
You're right. It's just matching for me. A separate parameter will be the best way to handle that.